ecl/network/v2/load_balancers/results.go

```diff
@@ -0,0 +1,151 @@
+type UpdateResult struct {
+	commonResult
+}
+
+// DeleteResult represents the result of a delete operation. Call its
+// ExtractErr method to determine if the request succeeded or failed.
+type DeleteResult struct {
+	eclcloud.ErrResult
+}
+
+// Interface represents a attached interfaces by Load Balancer.
+type Interface struct {
+	ID                  string `json:"id"`
+	IPAddress           string `json:"ip_address"`
+	Name                string `json:"name"`
+	NetworkID           string `json:"network_id"`
+	SlotNumber          int    `json:"slot_number"`
+	Status              string `json:"status"`
+	Type                string `json:"type"`
+	VirtualIPAddress    string `json:"virtual_ip_address"`
+	VirtualIPProperties string `json:"virtual_ip_properties"`
+}
+
+// Interface represents a attached interfaces by Load Balancer.
+type SyslogServer struct {
+	ID          string `json:"id"`
+	IPAddress   string `json:"ip_address"`
+	LogFacility string `json:"log_facility"`
+	LogLevel    string `json:"log_level"`
+	Name        string `json:"name"`
+	PortNumber  int    `json:"port_number"`
+	Status      string `json:"status"`
+}
+
+// LoadBalancer represents a Load Balancer. See package documentation for a top-level
+// description of what this is.
+type LoadBalancer struct {
+
+	// AdminPassword is admin's password
+	AdminPassword string `json:"admin_password"`
+
+	// AdminUsername is admin's username
+	AdminUsername string `json:"admin_username"`
+
+	// AvailabilityZone is one of the Virtual Server (Nova)’s availability zone.
+	AvailabilityZone string `json:"availability_zone"`
+
+	// Description is description
+	DefaultGateway string `json:"default_gateway"`
+
+	// Description is description
+	Description string `json:"description"`
+
+	// UUID representing the Load Balancer.
+	ID string `json:"id"`
+
+	// Attached interfaces by Load Balancer.
+	Interfaces []Interface `json:"interfaces"`
+
+	// LoadBalancerPlanID is the UUID of Load Balancer Plan.
+	LoadBalancerPlanID string `json:"load_balancer_plan_id"`
+
+	// Name of the Load Balancer.
+	Name string `json:"name"`
+
+	// The Load Balancer status.
+	Status string `json:"status"`
+
+	// Connected syslog servers.
+	SyslogServers []SyslogServer `json:"syslog_servers"`
+
+	// TenantID is the project owner of the Load Balancer.
+	TenantID string `json:"tenant_id"`
+
+	// User's password placeholder.
+	UserPassword string `json:"user_password"`
+
+	// User's username placeholder.
+	UserUsername string `json:"user_username"`
+}
+
+// LoadBalancerPage is the page returned by a pager when traversing over a collection
+// of load balancers.
+type LoadBalancerPage struct {
+	pagination.LinkedPageBase
+}
+
+// NextPageURL is invoked when a paginated collection of load balancers has reached
+// the end of a page and the pager seeks to traverse over a new one. In order
+// to do this, it needs to construct the next page's URL.
+func (r LoadBalancerPage) NextPageURL() (string, error) {
```

Comment:
The API of load balancer does not support paging. So this method should be removed.

Reply:
Got it, I fixed. I also fixed below files:
ecl/network/v2/load_balancer_interfaces/results.go
ecl/network/v2/load_balancer_plans/results.go
ecl/network/v2/load_balancer_syslog_servers/results.go